_targets.R

```diff
@@ -2,12 +2,14 @@ library(targets)
 library(tibble)
 library(tarchetypes)
 suppressPackageStartupMessages(library(tidyverse))
+library(retry)
 
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr",
                             "rnaturalearth", "cowplot", "lubridate",
-                            "leaflet", "leafpop", "htmlwidgets"))
+                            "leaflet", "leafpop", "htmlwidgets"),
+               error = "continue")
```

Comment:
Just a note - by adding `error = "continue"` here, it will apply to _all_ targets. If you want to just continue on if there is an error for a specific target, you can add `error = "continue"` as an argument to the individual `tar_target()` call.